torch/_dynamo/variables/tensor.py

```diff
@@ -708,6 +709,12 @@ def method_is_floating_point(self):
             return ConstantVariable.create(self.dtype.is_floating_point)
 
     def method_is_inference(self):
+        unimplemented_v2(
```

Comment:
we probably need an equivalent graph break for calls to `torch.is_inference_mode_enabled()`?

Reply:
jk didn't finish reading the PR...